### PR TITLE
monkey: update to 1.6.8.

### PR DESCRIPTION
**Pull request by Vaelatern**
File list for simpler review: https://gist.github.com/anonymous/75ac2c51ab2797ccaecd

### CODE REVIEW

**Comment by Vaelatern** [comment]
Note there is a `-devel` package without any `common/shlibs` entries, or even any versioned shlibs.

**Comment by Gottox** [comment]
> Note there is a -devel package without any common/shlibs entries, or even any versioned shlibs.

That's okay. We currently have no packages that link against monkey.

**Comment by ebfe** [comment]
Why did you remove the pkg-config file?

File: srcpkgs/monkey/template [hunk 1]
```diff
@@ -1,45 +1,40 @@
...
- /etc/monkey/plugins/auth/monkey.users"
+ /etc/monkey/plugins/*/*.conf
+ /etc/monkey/plugins/dirlisting/themes/*/*.theme
+ /etc/monkey/sites/default
+ /etc/monkey/monkey.conf
+ /etc/monkey/plugins.load
+ /etc/monkey/monkey.mime"
 
 system_accounts="httpd"
 httpd_descr="Monkey HTTP server"
```
Comment (Gottox) [inline srcpkgs/monkey/template]: I think we should remove the httpd description from all httpds. Currently the description of the user depends on the first httpd that was installed.

**Comment by Vaelatern** [comment]
@ebfe Please note that it isn't anywhere in the file list. I don't think it exists in this version.

File: srcpkgs/monkey/template (hunk 1 above)
Comment (Vaelatern) [inline srcpkgs/monkey/template]: Should I add that to this PR?

**Comment by ebfe** [comment]
@Vaelatern there's monkey.pc.in, so maybe an upstream bug that it doesn't get installed?

**Comment by ebfe** [comment]
I think the .so shouldn't be in the `-devel` package. These are plugins to be loaded by the webserver. If it has to be a separate package monkey-plugins would make more sense.

**Comment by Vaelatern** [comment]
Doesn't need to be a separate package at all. I'll poke at the pkg config file.

File: srcpkgs/monkey/template (hunk 1 above)
Comment (Gottox) [inline srcpkgs/monkey/template]: No. just wanted to mention it somewhere :)

Comment (Gottox) [inline srcpkgs/monkey/template]: we can push these changes this directly

**Pull request merged**